Src/FluentAssertions/Types/TypeSelector.cs

```diff
@@ -183,6 +183,32 @@ public TypeSelector ThatAreNotClasses()
         return this;
     }
 
+    /// <summary>
+    /// Determines wheter the type is a struct (a value type but not an Enum)
```

Comment:
🔧 It doesn't determine w_h_ether a type is struct. It filters them. Same with the other methods.